src/main.ts

```diff
@@ -48,7 +48,7 @@ interface BrazePocketHitsQuery {
   date: string;
 }
 
-app.get('/scheduled-items/:scheduledSurfaceID?date=:date', async (req, res) => {
+app.get('/scheduled-items/:scheduledSurfaceID', async (req, res) => {
```

Comment:
`scheduledSurfaceID` will be in `req.params`
`date` => `req.query`

Reply:
I'm fixing that in my PR here https://github.com/Pocket/braze-content-proxy/pull/16

Along with some other additions! @nina-py if you like, you can pull that PR or copy paste (to avoid commit clutter) some of the stuff.

Reply:
Thanks @Herraj! I've copied over the minimum I think I can get away with for this PR, and then the bulk of it will come through when your PR is merged.